src/shared/GitHub.Tests/GitHubHostProviderTests.cs

```diff
@@ -97,73 +97,28 @@ public async Task GitHubHostProvider_GetSupportedAuthenticationModes(string uriS
         }
 
 
-
-        [Fact]
-        public async Task GitHubHostProvider_GetSupportedAuthenticationModes_NotDotCom_OldInstanceNoPassword_ReturnsNone()
+        [Theory]
+        [InlineData("https://example.com", null, "0.1", false, AuthenticationModes.None)]
+        [InlineData("https://example.com", null, "0.1", true, AuthenticationModes.Basic)]
+        [InlineData("https://example.com", null, "100.0", false, GitHubConstants.DotComAuthenticationModes)]
```

Comment:
Can you replace the DotCom constant here with the explicit OAuth enumeration value please?
This is a test for the GitHub Enterprise mode discovery/logic and we don't want to make a link here to the DotCom modes constant.

```suggestion
        [InlineData("https://example.com", null, "100.0", false, AuthenticationModes.OAuth)]
```

Reply:
Sure. That's what was there before. This was intentionally without changing any logic.